--- a/wscript
+++ b/wscript
@@ -662,8 +662,13 @@ def configure(conf):
     Logs.pprint('CYAN', 'Configuring 32 bit host environment')
     # Copy 'local' to serve as the basis for '32bit':
     env_32bit = conf.env.derive().detach()
-    env_32bit.append_value('CFLAGS', '-m32')
-    env_32bit.append_value('LINKFLAGS', '-m32')
-    env_32bit.LINK_CC = 'gcc'
+    if sys.platform.startswith('linux'):
+        env_32bit.append_value('CFLAGS', '-m32')
+        env_32bit.append_value('LINKFLAGS', '-m32')
+        env_32bit.LINK_CC = 'gcc'
+        env_32bit.append_value('CFLAGS', '-O2')
+        env_32bit.append_value('CFLAGS', '-Wno-error=stringop-truncation')
+        env_32bit.CC = 'gcc'
+        env_32bit.CXX = 'g++'
     conf.all_envs['32bit'] = env_32bit
     conf.set_env(prev_env)
