--- a/waftools/gitinfo.py
+++ b/waftools/gitinfo.py
@@ -1,6 +1,7 @@
 # SPDX-FileCopyrightText: 2024 Google LLC
 # SPDX-License-Identifier: Apache-2.0

+import os
 import re

 import waflib.Context
@@ -8,20 +9,9 @@ import waflib.Logs


 def get_git_revision(ctx):
-    commit = ctx.cmd_and_log(
-        ["git", "rev-parse", "--short", "HEAD"], quiet=waflib.Context.BOTH
-    ).strip()
-    timestamp = ctx.cmd_and_log(
-        ["git", "log", "-1", "--format=%ct", "HEAD"], quiet=waflib.Context.BOTH
-    ).strip()
-
-    try:
-        tag = ctx.cmd_and_log(
-            ["git", "describe", "--dirty"], quiet=waflib.Context.BOTH
-        ).strip()
-    except Exception:
-        tag = "v9.9.9-dev"
-        waflib.Logs.warn(f"Git tag not found, using {tag}")
+    tag = os.environ.get("PEBBLE_GIT_TAG", "v0.0.0")
+    commit = os.environ.get("PEBBLE_GIT_COMMIT", "nix")
+    timestamp = os.environ.get("PEBBLE_GIT_TIMESTAMP", "0")

     # Validate that git tag follows the required form:
     # See https://github.com/pebble/tintin/wiki/Firmware,-PRF-&-Bootloader-Versions