--- a/wscript
+++ b/wscript
@@ -543,11 +543,6 @@ def configure(conf):
     conf.setenv('stored_apps', base_env)
     conf.recurse('stored_apps')

-    # Confirm that requirements-*.txt and requirements-osx-brew.txt have been satisfied.
-    import tool_check
-    tool_check.tool_check()
-
-
 def _run_remote_suite(ctx, suite):
     # PEBBLESDK_TEST_ROOT must be defined in order to initiate integration tests
     try:
