--- a/libpebble2/util/bundle.py
+++ b/libpebble2/util/bundle.py
@@ -42,6 +42,7 @@ class PebbleBundle(object):
         'diorite': ('diorite/', 'aplite/', ''),
         'emery': ('emery/', 'basalt/', ''),
         'flint': ('flint/', 'diorite/', 'aplite/', ''),
+        'gabbro': ('gabbro/',),
     }
 
     MAX_COMPATIBILITY_VERSIONS = {
--- a/libpebble2/util/hardware.py
+++ b/libpebble2/util/hardware.py
@@ -31,6 +31,7 @@ class PebbleHardware(object):
     ROBERT_BB2 = 0xF7
     SILK_FLINT = 0xF6
     SNOWY_ROBERT = 0xF5
+    SPALDING_GABBRO_BB = 0xF2
 
     PLATFORMS = {
         UNKNOWN: 'unknown',
@@ -61,6 +62,7 @@ class PebbleHardware(object):
         ROBERT_BB2: 'emery',
         SILK_FLINT: 'flint',
         SNOWY_ROBERT: 'emery',
+        SPALDING_GABBRO_BB: 'gabbro',
     }
 
     @classmethod
